include/mango_v3.hpp

```diff
@@ -140,6 +142,39 @@ struct MangoAccount {
   uint8_t padding[5];
 };
 
+class MangoAccount {
+  // Creating a MangoAccount might involve non-trivial work like calling RPC
+  // methods, so we provide explict `MangoAccount::from()` overloaded set of
+  // functions and make the public constructor un-callable.
+  struct NotPubliclyConstructible {};
```

Comment:
dont buy the argument, same could be achieved by just setting the default constructor private, as even rpc methods only use exceptions for error handling, they are fine to be called from constructors.